docker-compose.yml

```diff
@@ -35,7 +35,7 @@ services:
       - rails-tmp:/app/samvera/hyrax-webapp/tmp
 
   chrome:
-    image: selenium/standalone-chrome:3.141
```

Comment:
this version scheme is still in use, and we might want to keep using it to make it easier to match the version on the ruby end: https://hub.docker.com/r/selenium/standalone-chrome/tags?page=1&ordering=last_updated

Reply:
good call. in the meanwhile, i'm able to confirm that i could run the feature specs in docker-compose with this PR.